pkg/windows/windows.go

```diff
@@ -374,13 +384,42 @@ func (vm *windows) Configure() error {
 	if err := vm.transferFiles(); err != nil {
 		return errors.Wrap(err, "error transferring files to Windows VM")
 	}
+	if err := vm.configureContainerd(); err != nil {
+		return errors.Wrapf(err, "error configuring containerd")
+	}
 	if err := vm.ConfigureWindowsExporter(); err != nil {
 		return errors.Wrapf(err, "error configuring Windows exporter")
 	}
 
 	return vm.runBootstrapper()
 }
 
+// configureContainerd starts the Windows containerd service
+func (vm *windows) configureContainerd() error {
+	// set Windows defender exclusions for containerd
+	setExclusionCmd := "Add-MpPreference -ExclusionProcess " + containerdPath
+	out, err := vm.Run(setExclusionCmd, true)
+	if err != nil {
+		vm.log.Info("setting Windows defender exclusion failed", "command", setExclusionCmd,
```

Comment:
yes. This is kind of a must until containerd fix "downloadtimeexcced timeout"  issue. Without this, image download will take a much longer time. So we can make this an error message even though we don't stop containerd configuration.

Reply:
>Not sure error logging would helpful for the user here, the error is being returned later which should lead to reconcile here.

Given the error is being returned later, I suggest printing the command information with a lower verbosity level (debug), otherwise no logging at all.